tests/e2e/.mocharc.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
   retries: 4,
   bail: true,
   exit: true,
-  recursive: true,
```

Comment:
Should we keep this?

Reply:
It was a duplicate key I removed :) Line 4 already has the same key

Reply:
Nice :)